BasicTVPlayback/src/main/java/com/blendvision/player/basic/playback/tvsample/MainActivity.kt

```diff
@@ -0,0 +1,90 @@
+package com.blendvision.player.basic.playback.tvsample
+
+import android.os.Bundle
+import android.view.KeyEvent
+import androidx.fragment.app.FragmentActivity
+import androidx.lifecycle.ViewModelProvider
+import com.blendvision.player.basic.playback.tvsample.databinding.ActivityMainBinding
+import com.kkstream.playcraft.paas.player.common.callback.ControlStateEventListener
+import com.kkstream.playcraft.paas.player.stb.CustomUniTVFragmentInterface
+import com.kkstream.playcraft.paas.player.stb.StateType
+import com.kkstream.playcraft.paas.player.stb.UniTvFragment
+
+class MainActivity : FragmentActivity() {
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var uniTvFragment: UniTvFragment
+    private lateinit var viewModel: MainViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        viewModel = ViewModelProvider(this)[MainViewModel::class.java]
+
+        uniTvFragment = UniTvFragment()
+
+        supportFragmentManager.beginTransaction()
+            .replace(R.id.fragmentContainerView, uniTvFragment)
+            .commit()
+
+    }
+
+    private fun setupUniTvFragment() {
+        val unPlayer = viewModel.setupPlayer(this)
```

Comment:
Did you mean `uniPlayer` here?

For a sample & real usage, "uni" prefix is not necessary and we may omit it, and the scope is small, simply `player` is OK.